### PR TITLE
Fix selenium img

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
rerun selenium-tests

File: scripts/ci/cleanup.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ function cleanup() {
   cleanup_items ovsdb "ovs-vsctl del-br"
   cleanup_items docker "docker_rm"
   cleanup_items lxd "lxc delete --force"
-  cleanup_items docker-images "docker rmi -f"
+  #cleanup_items docker-images "docker rmi -f"
```
Comment (hunchback) [inline scripts/ci/cleanup.sh]: Better to remove line

Comment (hunchback) [reply scripts/ci/cleanup.sh]: Better to remove line

File: tests/selenium_test.go [hunk 2]
```diff
@@ -158,6 +158,9 @@ func TestPacketInjectionCapture(t *testing.T) {
 		return
 	}
 
+	// wait a bit to be sure that the capture is really started because of slow systems.
+	time.Sleep(5 * time.Second)
```
Comment (hunchback) [inline tests/selenium_test.go]: Rather retry the Gremlin statement bellow

File: scripts/ci/cleanup.sh [hunk 3]
```diff
@@ -76,7 +76,7 @@ function snapshot() {
   snapshot_items intf $ext "ip -o link show | awk -F': ' '{print \$2}' | cut -d '@' -f 1"
   snapshot_items ovsdb $ext "ovs-vsctl list-br"
   snapshot_items docker $ext "docker ps -a -q"
-  snapshot_items docker-images $ext "docker images -a -q"
+  #snapshot_items docker-images $ext "docker images -a -q"
```
Comment (hunchback) [inline scripts/ci/cleanup.sh]: Better to delete

File: tests/selenium_test.go (hunk 2 above)
Comment (safchain) [reply tests/selenium_test.go]: I don't think so, as it is to inject and I don't want to inject twice or more

Comment (hunchback) [reply tests/selenium_test.go]: I am fine with the above if it works even on very-slow systems ... if this is not the case I would still recommend polling (such as 10 retry / 1s delay) ... is fast on fast systems but will still wait a while to give slow systems a chance to come up.

File: scripts/ci/cleanup.sh (hunk 3 above)
Comment (safchain) [reply scripts/ci/cleanup.sh]: will do once I will have something really working

**Comment by safchain** [comment]
rerun selenium-tests

**Comment by safchain** [comment]
rerun unit-tests

**Comment by safchain** [comment]
rerun selenium-tests

**Comment by safchain** [comment]
rerun selenium-tests

**Comment by safchain** [comment]
rerun selenium-tests

**Pull request merged**